app/src/main/java/com/kickstarter/ui/toolbars/AlphaBehavior.java

```diff
@@ -0,0 +1,35 @@
+package com.kickstarter.ui.toolbars;
+
+import android.content.Context;
+import android.support.design.widget.AppBarLayout;
+import android.support.design.widget.CoordinatorLayout;
+import android.util.AttributeSet;
+import android.view.View;
+
+
+public class AlphaBehavior<V extends View> extends CoordinatorLayout.Behavior<V> {
+
+  public AlphaBehavior() {
+  }
+
+  public AlphaBehavior(final Context context, final AttributeSet attrs) {
```

Comment:
nice use of a custom view class!!!!!!! only comment here are to annotate the method params `@NonNull`